app/javascript/components/individual/SplitViewContainer.jsx

```diff
@@ -22,19 +22,19 @@ const SplitViewContainer = props => (
       {props.leftComponent}
     </div>
 
-    <div key="right"
-      style={{
-        width: "960px",
-        height: "100vh",
-        display: "inline-block",
-        overflow: "scroll",
-        paddingLeft: "10px",
-        borderLeft: "1px solid gray"
-      }}
-    >
-      {props.rightComponent}
-    </div>
-  </div>
+		<div key="right"
```

Comment:
fix spacing here to 2 spaces

Reply:
Also pull from master, i think there may be small merge conflicts here.

Reply:
done @ both